sechub-notification/src/main/java/com/daimler/sechub/domain/notification/superadmin/InformAdminsThatSchedulerJobProcessingHasBeenDisabledService.java

```diff
@@ -38,8 +38,8 @@ public void notify(String baseUrl) {
 
 	private String createEmailContent(String baseUrl) {
 		StringBuilder emailContent = new StringBuilder();
-		emailContent.append("Scheduler job processing has been disabled at sechub for environment (base url):").append(baseUrl).append("\n");
-		emailContent.append("WARNING: Users can still add new Jobs, but will wait for execution until scheduler job processing will be enabled again!");
+		emailContent.append("Scheduler job processing has been disabled at SecHub for environment (base url): " + baseUrl + "\n\n");
```

Comment:
Please look at https://www.baeldung.com/java-strings-concatenation
where mixing up append and + operatior is mentioned as bad practice - as it is...

Reply:
changed all "+" into .append()